roles/marathon/templates/mesos-consul.json.j2

```diff
@@ -0,0 +1,21 @@
+{
+  "args": [
+    "--zk=zk://zookeeper.service.{{ consul_dns_domain }}:2181/mesos",
+    "--registry-ssl",
+    "--registry-ssl-verify=false",
```

Comment:
I'm going to be rebasing the changes from this branch on top of current master. I'll get that change in there, thanks for the catch.